python/mxnet/context.py

```diff
@@ -80,9 +97,21 @@ def __exit__(self, ptype, value, trace):
 
 
 def cpu(device_id=0):
-    """Return a CPU context.
+    """Returns a CPU context.
 
     This function is a short cut for ``Context('cpu', device_id)``.
+    cpu() is usually the default context for many operations when no context is specified.
```

Comment:
Perhaps we can put cpu() in backticks because it's code (a function call) and also can we avoid starting the sentence with it? Change to:

"For most operations, when no context is specified, the default context is `cpu()`"